### PR TITLE
User Managment into spaceship

### PR DESCRIPTION
**Pull request by hjanuschka**
bildschirmfoto 2017-01-25 um 21 53 05

adds the ability to manage itc users via spaceship.

API:

# Get all members

```ruby
Spaceship::Members.all #returns array of Spaceship::Tunes::Member
```

bildschirmfoto 2017-01-25 um 14 18 07

# find a specific member

```ruby
Spaceship::Members.find("helmut@januschka.com")` -> `nil` or `Spaceship::Tunes::Member
```

Base: `member = Spaceship::Members.all.first`

# delete a member

```ruby
member.delete!
```

# resend invitation

```ruby
member.resend_invitation
```

# Create a Member

> default, admin with all apps.

```ruby
Spaceship::Members.create!(
  firstname: "heli1",
  lastname: "lastname2", 
  email_address: "hjanuschka+spaceship@gmail.com"
)
```

# Create member with Role and all apps

```ruby
Spaceship::Members.create!(
  firstname: "heli1",
  lastname: "lastname2", 
  email_address: "hjanuschka+spaceship@gmail.com"
  roles: ["developer"]
)
```

# Create Member with role and specific app's

```ruby
Spaceship::Members.create!(
  firstname: "heli1",
  lastname: "lastname2", 
  email_address: "hjanuschka+spaceship@gmail.com"
  roles:["developer"]
  apps:["123123"]
)
```

### CODE REVIEW

File: spaceship/lib/spaceship/tunes/member.rb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        'lastName.value' => :lastname,
+        'userName' => :username,
+        'dsId' => :user_id
+      )
+
+      class << self
+        def factory(attrs)
+          self.new(attrs)
+        end
+      end
```
Comment (KrauseFx) [inline spaceship/lib/spaceship/tunes/member.rb]: Please add spaces after an `end`

**Comment by hjanuschka** [comment]
@KrauseFx  feedback addressed. ready for review

File: spaceship/spec/tunes/members_spec.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+describe Spaceship::Tunes::Members do
+  before { Spaceship::Tunes.login }
+  before { TunesStubbing.itc_stub_members }
+
+  describe "members" do
+    it "should return a list with members" do
+      members = Spaceship::Members.all
+      expect(members.length).to eq(3)
+    end
```
Comment (KrauseFx) [inline spaceship/spec/tunes/members_spec.rb]: Can you always add new lines after an `end` block unless there is another `end` block following?

**Review by KrauseFx — APPROVED** [review]

**Comment by KrauseFx** [comment]
This looks awesome, I added one more comment, once that's updated, feel free to click the merge button  🚀

**Comment by hjanuschka** [comment]
🤘 thx!!!

**Pull request merged**